docs/scos/dev/front-end-development/migration-guide-remove-support-of-ie11.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Migration guide - Remove support of IE11
+description: Use this guide to migrate the project from Yarn to NPM v8.
+template: concept-topic-template
+---
+
+This document provides instructions for removing support of IE11 in your Spryker project.
+
+## Overview
+
+Microsoft dropped the support of IE11 in June 2022. So it has to be removed in Spryker.
+
+*Estimated migration time: 1h*
+
+## Update modules
+
+Run the next command to update module versions:
+```bash
+composer update spryker/zed-ui spryker-shop/shop-ui
+```
+
+## Update configuration files
+
+`.browserslistrc`:
+
+Remove IE11 usage:
+```text
+IE 11
+```
+
+`frontend/configs/development.js`: 
+
+1. Remove `buildVariantSettings` variable declaration and usage:
+```js
+const { buildVariantSettings } = require('../settings');
+...
+const { buildVariant, isES6Module } = buildVariantSettings;
+```
+
+2. Remove `es6PolyfillTs` variable declaration and usage:
+```js
+const es6PolyfillTs = await findAppEntryPoint(appSettings.find.shopUiEntryPoints, './es6-polyfill.ts');
+...
+'es6-polyfill': es6PolyfillTs,
+```
+
+3. Remove browsers support as it specified in the `.browserslistrc` file:
+```js
+browsers: [
+    '> 1%',
+    'ie >= 11',
+],
+```
+
+```js
+plugins: [
+    autoprefixer({
+        'browsers': ['> 1%', 'last 2 versions']
+    })
+]
+// should be
+plugins: [require('autoprefixer')],
```

Comment:
You didn't mention that the require at the top should be removed so they will have two `require('autoprefixer')`